### ISSUE TITLE
Cardboard Freezes after Start-Up

### ISSUE THREAD

**Issue by mdejong166** [googlevr/cardboard#477]
### SPECIFIC ISSUE ENCOUNTERED
I followed the quickstart-guide from Google to run the cardboard-app. After hitting build & run, the app starts on my Android Phone, the Unity Intro shows, the HelloCardboard-Scene shows up but the camera doesn't move and I only have one screen (instead of the two for the Cardboard-goggle). Logcat tells me to "Please initialize Cardboard XR loader before calling this function.". 

Adding an if-statement to the CardboardStartup.cs on Api.UpdateScreenParams(); did not work. 

### SMARTPHONE (please complete the following information):
- Device: Pixel 6
- Android 14

### STEPS TO REPRODUCE THE ISSUE
1. Follow the Quickstart-Guide
2. Hit Build and Run

### EXPECTED BEHAVIOR
I'd expect that when the app starts on my smartphone, I'd be able to look around by moving the smartphone. Also I'd expect the screen to be split in two to be compatible with the cardboard-goggle.

### VERSIONS USED
Cardboard XR Plugin 1.26.0

**If you are using Cardboard XR Plugin:**
I tried with Unity Versions:
- 2022.3.50f1
- 2021.3.45f1
- 6000.0.23f1 (this one didn't get as far as the other ones. It crashed before the Unity-Logo loaded)

- Select in which graphics API you can reproduce the bug.
   - [ x ] OpenGL ES3
   - [ x ] Vulkan

### LINK TO A REPOSITORY WITH CODE TO REPRODUCE THE BUG
https://github.com/googlevr/cardboard

### SCREENSHOTS
![image](https://github.com/user-attachments/assets/19809f97-40fc-4f1a-a868-cb80d4fdd23a)

### ADDITIONAL CONTEXT
I really hope, I phrased this well. Thanks a lot in advance for reading and caring! :)

**Comment by DARK-Shadw** [comment]
Yes facing the same issue the vr works fine at the splash screen for unity but then it freezes if any fix is found please let me know. Also is it possible to try out an older version on cardboard?

**Comment by lneumarkt** [comment]
Thank you @mdejong166 and @DARK-Shadw for reaching out!

We’ve tried to replicate the problem using different Android devices. But unfortunately we couldn’t reproduce the bug.

In order to allow us to reproduce this issue, could you please answer the following items?

- Did you experiment the reported behavior on a different device?
- Could you please share your device's fingerprint?
  - You can get it by running the following command in the terminal with the device connected to your computer.
  `adb shell getprop ro.build.fingerprint`
- Please share the required steps to expose the issue with us being as precise as possible so as to be sure that we get the same configuration as you do. Even if you just followed the quickstart, please list the steps you performed.
- A simple Unity project where this issue is depicted would also be very helpful.

Thanks!

**Comment by Mandelbrow** [comment]
@mdejong166 Did you follow all steps in the Google Cardboard Quickstart? 

For example this step:
Navigate to Project Settings > XR Plug-in Management.
  1.  Select Cardboard XR Plugin under Plug-in Providers.
